auslib/db.py

```diff
@@ -1539,6 +1539,27 @@ def _buildIDMatchesRule(self, ruleBuildID, queryBuildID):
             return True
         return string_compare(queryBuildID, ruleBuildID)
 
+    def _processBooleanRuleString(self, rule_str):
+        return [rule.split('&&') for rule in rule_str.split(',')]
```

Comment:
I think this whole section would read better if you put this in _simpleBooleanMatchesRule instead of its own function. If it was multiple lines of code it might be worth doing, but currently this just seems to add indirection with no benefit.